app/Http/Controllers/EpisodeManager.php

```diff
@@ -117,9 +138,58 @@ public function uploadFileToS3(Request $request)
     {
         $fileName = time() . '.' . $request->podcast->getClientOriginalExtension();
         $s3 = Storage::disk('s3');
+        
         //large files
         $s3->put($fileName, fopen($request->podcast, 'r+'));
 
         return $s3->getDriver()->getAdapter()->getClient()->getObjectUrl('suyabay', $fileName);
     }
+
+    public function delete(Request $request)
+    {
+        $episode_id  = $request['episode_id'];
+        $episode     = $this->episodeRepository->findEpisodeWhere("id", $episode_id)->delete();
+        
+        if ($episode  === 1)
+        {
+            $data = [
+                "status"    => 200,
+                "message"   => "Episode successfully deleted"
+            ];
+        }
+        
+        if ($episode  === 0)
+        {
+            $data = [
+                "status"    => 401,
+                "message"   => "episode can not be deleted"
+            ];
+        }
+
+        return $data;
+    }
+
+    public function updateEpisodeStatus (Request $request)
+    {
+        $episode_id     = $request['episode_id'];
+        $episode        = $this->episodeRepository->findEpisodeWhere("id", $episode_id)->update(['status' => 1]);
+
+        if ($episode  === 1)
+        {
+            $data = [
+                "status"    => 200,
+                "message"   => "Episode successfully updated"
+            ];
+        }
+        
+        if ($episode  === 0)
```

Comment:
The opening brace after `if` should be on the same line. Please, follow PS2-2 guidelines, you should know this by now!